src/components/tooltip/tooltip.tsx

```diff
@@ -82,12 +82,12 @@ export class Tooltip {
     @Prop({ reflect: true })
     public maxlength?: number = DEFAULT_MAX_LENGTH;
 
-    @State()
-    private open: boolean;
-
     @Element()
     private host: HTMLLimelTooltipElement;
 
+    @State()
+    private open: boolean;
```

Comment:
In my world I try to make public properties or functions on top then the others in the following order public, protected, private. Here both the `@Element()` and `@State()` are private. What make you change this if I may ask

Reply:
Ok that is fine.